test/unit/test_events.py

```diff
@@ -5,18 +5,20 @@
 from dbt.events.functions import event_to_serializable_dict
 from dbt.events.types import *
 from dbt.events.test_types import *
-from dbt.events.base_types import Event
+from dbt.events.base_types import Event, InfoLevel
 from dbt.events.stubs import _CachedRelation, BaseRelation, _ReferenceKey, ParsedModelNode
 from importlib import reload
 import dbt.events.functions as event_funcs
 import dbt.flags as flags
+from dbt.lazy import Lazy
```

Comment:
I'm getting a `ModuleNotFoundError: No module named 'dbt.lazy'` error when I run my tests. I found this pattern in other unit tests, what am I doing wrong here?

Reply:
what tests have dbt.lazy? if i do a search in finder I find no dbt.lazy in core

Reply:
local virtual environment issue.